src/test/java/org/commcare/fixtures/test/IndexedFixtureTests.java

```diff
@@ -68,4 +75,48 @@ public void errorOnMaliciousSchemaTest() throws XPathSyntaxException, Unfullfill
             XmlPullParserException, IOException, InvalidStructureException {
         ParseUtils.parseIntoSandbox(getClass().getResourceAsStream("/malicious-indexed-fixture.xml"), sandbox, true);
     }
+
+    @Test
+    public void testPartialQueryLoads() throws XPathSyntaxException, UnfullfilledRequirementsException,
```

Comment:
These 2 tests are identical except for the file path passed into `getResourceAsStream()`. Can you refactor them to use a shared helper?